lms/envs/devstack.py

```diff
@@ -268,6 +268,22 @@ def should_show_debug_toolbar(request):
     'JWT_AUDIENCE': 'lms-key',
 })
 
+
+############## Settings for ACE ####################################
+ACE_ENABLED_CHANNELS = [
+    'file_email'
+]
+ACE_ENABLED_POLICIES = [
+    'bulk_email_optout'
+]
+ACE_CHANNEL_SAILTHRU_DEBUG = True
+ACE_CHANNEL_SAILTHRU_TEMPLATE_NAME = 'Automated Communication Engine Email'
+ACE_CHANNEL_SAILTHRU_API_KEY = None
+ACE_CHANNEL_SAILTHRU_API_SECRET = None
+
+ACE_ROUTING_KEY = LOW_PRIORITY_QUEUE
```

Comment:
Is it intentional to merge these changes?  These settings are already set as the same in `lms/envs/common.py`.

Reply:
devstack get's overridden with yaml settings that are empty, so this makes it so that devstack works correctly by default.

Reply:
fixed